content/en/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -0,0 +1,113 @@
+---
+reviewers:
+- dims
+- 44past4
+title: System Logs
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+
+System component logs can present you a most detailed view of things happening in cluster, which can be very useful for debugging.
+Depending on configured verbosity, logs can provide very different detail.
+From simple information about errors in within component, up to step-by-step traces of events (e.g. HTTP access logs, pod state changes, controller actions, scheduler decisions).
+
+<!-- body -->
+
+## Klog
+
+Kubernetes components use [klog] logging library.
+[klog] can be configured via command line flags exposed by components.
+You can find their list in [command-line-tools-reference]. Example of klog format:
+
+```
+I1025 00:15:15.525108       1 httplog.go:79] GET /api/v1/namespaces/kube-system/pods/metrics-server-v0.3.1-57c75779f-9p8wg: (1.512ms) 200 [pod_nanny/v0.0.0 (linux/amd64) kubernetes/$Format 10.56.1.19:51756]
+```
+
+[command-line-tools-reference]: /docs/reference/command-line-tools-reference
+[klog]: https://github.com/kubernetes/klog
+
+### Structured Logging (ALPHA)
+
+Structured logging is a effort to introduce a uniform structure in log messages allowing for easy extraction of information, making logs easier and cheaper to store and process.
+New message format is backward compatible and enabled by default.
+
+Format of structured logs:
+```
+<klog header> "<message>" <key1>="<value1>" <key2>="<value2>" ...
+```
+
+Example:
+```
+I1025 00:15:15.525108       1 controller_utils.go:116] "Pod status updated" pod="kube-system/kubedns" status="ready"
+```
+
+**WARNING** AS structured format is an ALPHA feature, details like exact format or value serialization are a subject to change.
+
+**WARNING** Currently only a small portion of logs utilize structured message format, migration to currently ongoing and is expect to take very long time before every log call is migrated. If you intend to parse logs, please make sure to handle normal messages.
+
+### JSON log format (ALPHA)
+
+By providing `--logging-format=json` flag to commandline we can change format of logs written by Kubernetes System components.
```

Comment:
I'm not sure about working here "flag returns structured log messages". Flag doesn't do any returning, it just configures component to write json instead of previously mentioned klog format.

Reply:
@serathius I think we can use "flag returns" safely to infer behavior of the associated command.